packages/desktop-client/src/components/reports/DateRange.tsx

```diff
@@ -34,43 +36,59 @@ export function DateRange({ start, end, type }: DateRangeProps): ReactElement {
   } else {
     return (
       <Text style={{ ...styles.mediumText, color: theme.errorText }}>
-        There was a problem loading your date range
+        <Trans>There was a problem loading your date range</Trans>
       </Text>
     );
   }
 
+  const formattedStartDate = d.format(startDate, 'MMM yyyy');
+  const formattedEndDate = d.format(endDate, 'MMM yyyy');
+  let typeOrFormattedEndDate: string;
+
+  if (type && ['budget', 'average'].includes(type)) {
+    typeOrFormattedEndDate = type;
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏
```suggestion
    typeOrFormattedEndDate = type === 'budget' ? 'budgeted' : type;
```